Description: Install private executables in /usr/libexec/vfu
 This fixes lintian executable-in-usr-lib
 .
 vfu (4.18-2) unstable; urgency=medium
Forwarded: not-needed
Last-Update: 2020-07-24

--- vfu-4.18.orig/install
+++ vfu-4.18/install
@@ -11,22 +11,22 @@ if [ ! -e rx/rx_auto ]; then
   exit;
 fi
 
-echo "Press ENTER to install vfu in /usr/local/bin and rx_* in /usr/lib/vfu"
+echo "Press ENTER to install vfu in /usr/local/bin and rx_* in /usr/libexec/vfu"
 echo "Or press Ctrl+C to cancel"
 read
 
 cp vfu/vfu /usr/local/bin
-mkdir /usr/lib/vfu
-cp rx/rx_* /usr/lib/vfu
+mkdir /usr/libexec/vfu
+cp rx/rx_* /usr/libexec/vfu
 
 cp vfu.1 /usr/local/man/man1
 
-cp vfu.conf /usr/lib/vfu
+cp vfu.conf /usr/libexec/vfu
 
 cd /usr/local/bin
 chmod 755 vfu
 
-cd /usr/lib/vfu
+cd /usr/libexec/vfu
 chmod 755 rx_*
 
 echo "done."
--- vfu-4.18.orig/vfu/vfuarc.cpp
+++ vfu-4.18/vfu/vfuarc.cpp
@@ -27,7 +27,7 @@
     archive_path = ""; /* cannot have path when recursing archive */
 
   VString s;
-  s = "rx_auto ";
+  s = "/usr/libexec/vfu/rx_auto ";
   s += ( a_recursive ) ? "v" : "l";
   s += " '" + archive_name + "' ";
   s += " '" + archive_path + "' ";
@@ -127,7 +127,7 @@
   VString fn = files_list_get(FLI)->full_name();
 
   VString s;
-  s = "/usr/lib/vfu/rx_auto x \"";
+  s = "/usr/libexec/vfu/rx_auto x \"";
   s += work_path;
   s += archive_name;
   s += "\" ";
@@ -160,7 +160,7 @@
   VString fn = files_list_get(FLI)->full_name();
 
   VString s;
-  s = "/usr/lib/vfu/rx_auto x \"";
+  s = "/usr/libexec/vfu/rx_auto x \"";
   s += work_path;
   s += archive_name;
   s += "\" ";
@@ -222,7 +222,7 @@
   chmod( tmpfile, S_IRUSR|S_IWUSR );
 
   VString s;
-  s = "/usr/lib/vfu/rx_auto x \"";
+  s = "/usr/libexec/vfu/rx_auto x \"";
   s += work_path;
   s += archive_name;
   s += "\" @";
--- vfu-4.18.orig/vfu/vfusetup.h
+++ vfu-4.18/vfu/vfusetup.h
@@ -53,7 +53,7 @@
 #endif
 
 #ifndef FILENAME_CONF_GLOBAL2
-#define FILENAME_CONF_GLOBAL2 "/usr/lib/vfu/" FILENAME_CONF
+#define FILENAME_CONF_GLOBAL2 "/usr/libexec/vfu/" FILENAME_CONF
 #endif
 
 #define RX_TEMP_LIST "RX_TEMP_LIST"
